cabinet/cabinet.py

```diff
@@ -56,6 +56,20 @@ def get_all(self):
     def get(self, name):
         return self._vault.get(name)
 
+    def get_by_tags(self, tags=[]):
```

Comment:
I didn't know that! I'm on it, your example exposes a nice habit!
Just asking: It shouldn't be a problem if `tags` is read only, right?.